metagraph/docs/source/quick_start.rst

```diff
@@ -149,12 +155,12 @@ half of the space.
 .. TODO: note that canonical graphs must not be used with row-diff<*> annotations and always must be primarized
 
 Graph cleaning
-""""""""""""""
+^^^^^^^^^^^^^^
 
-For removing sequencing noise, there are graph cleaning and k-mer
-filtering procedures implemented in MetaGraph. These are based on the assumption that
-k-mers with a relatively low abundance in the input data are likely due to sequencing errors, and
-hence should be dropped to keep the k-mer index free of the non-existent k-mers.
+For removing sequencing errors, MetaGraph provides routines for graph cleaning and k-mer
+filtering. These are based on the assumption that
+k-mers with relatively low abundance (k-mer counts) in the input data are likely due to sequencing errors, and
+hence should be dropped to keep the graph free of the non-existent k-mers.
```

Comment:
```suggestion
hence should be dropped to reduce noise in the final graph.
```
The previous statement was too strong, we never guarantee that our cleaning strategy is perfect.

Reply:
Strictly speaking, the new statement is wrong too.

Well, this is not a theorem, anyway.

Reply:
I don't like that it says there is noise in the graph. There is no noise. It contains all k-mers from the input.

Reply:
It depends on what you choose as your baseline: 1) the true DNA sequence of the individual you wanted to sequence? 2) the true DNA sequences of whatever DNA is in your sample?, or 3) the measured k-mers from the sequencing experiment.

Maybe we should talk about this a bit. Currently, the term "non-existent" is not defined.

Reply:
When you say that there is no noise in the graph, then it sounds like you're considering 3) to be your baseline, in which case, this procedure is removing signal.

Reply:
It's a curation algorithm that operates on the input graph. There is no ambiguity. Ok, both statements were inaccurate. I'll write something different.